### ISSUE TITLE
[Feature]: Post draft cache

### ISSUE THREAD

**Issue by DanGN0** [PretendoNetwork/juxtaposition-ui#92]
### Checked Existing

- [X] I have checked the repository for duplicate issues.

### What feature do you want to see added?

An offline draft/cache function for Juxtaposition posts, where it automatically saves in cache your latest draft, for both text and drawing. 

### Why do you want to have this feature?

Because it's happened more than once that I'd have drawn very detailed pieces only for my wifi to temporarily cut out and thus blocking the entire app, forcing one to lose everything even though it had nothing to do with connection to the server. (Where a pop-up shows up saying to either reload or close Juxtaposition.)

TLDR: So it doesn't wipe out drafts when Internet connection cuts temporarily (especially knowing how... unreliable a 3DS' Internet connection can be).

**Comment by jonbarrow** [comment]
I'll approve this as something to look into, but I have my doubts at this becoming a thing. The browsers Miiverse uses on these consoles has very limited memory, and trying to store multiple draft posts may not be feasible
